### PR TITLE
Support for Kubernetes v1.22

### PR DESCRIPTION
**Pull request by timuthy**
**How to categorize this PR?**

/area open-source usability
/kind enhancement
/priority 3
/platform aws
/exp intermediate
/topology garden seed shoot

**What this PR does / why we need it**:
This PR adds support for 1.22 to the extension.

**Which issue(s) this PR fixes**:
Part of gardener/gardener#4363

**Special notes for your reviewer**:
* :white_check_mark: Depends on gardener/gardener#4562
* I have successfully validated the functionality as follows:
  * :white_check_mark: Create new clusters with versions < 1.22
  * :white_check_mark: Create new clusters with version  = 1.22
  * :white_check_mark: Upgrade old clusters from version 1.21 to version 1.22
  * :white_check_mark: Delete clusters with versions < 1.22
  * :white_check_mark: Delete clusters with version  = 1.22

**Release note**:

```feature user
The AWS extension does now support shoot clusters with Kubernetes version 1.22. You should consider the [Kubernetes release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) before upgrading to 1.22. 
```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -57,7 +62,7 @@ images:
 - name: csi-attacher
   sourceRepository: github.com/kubernetes-csi/external-attacher
   repository: k8s.gcr.io/sig-storage/csi-attacher
-  tag: "v2.2.0"
+  tag: "v3.3.0"
```
Comment (rfranzke) [inline charts/images.yaml:65]: Is this version compatible with 1.18-1.21? :)

**Review by rfranzke — COMMENTED** [review]
Looks good, just a minor question

File: charts/images.yaml (hunk 1 above)
Comment (timuthy) [reply charts/images.yaml:65]: Yep :) https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.3.0 see "Minimum Kubernetes version". I also tested it.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**